Github-Issue-Guide.md

```diff
@@ -0,0 +1,68 @@
+# Filing an Issue on Oppia's GitHub: A Step-by-Step Guide for Non-Developers
+
+## Table of Contents
+- [Create a GitHub Account](#create-a-github-account)
+- [Access Oppia's GitHub Repository](#access-oppias-github-repository)
+- [File a New Issue](#file-a-new-issue)
+- [Engage in the Issue Discussion](#engage-in-the-issue-discussion)
+- [Conclusion](#conclusion)
+
+Welcome to Oppia's GitHub issue filing guide for non-developers! This step-by-step guide will help you navigate the process of creating a GitHub account and filing an issue on Oppia's GitHub repository. Let's get started!
+
+## Create a GitHub Account
+
+1. Visit the GitHub website at [https://github.com](https://github.com).
+
+2. Click on the "Sign up" button to create a new GitHub account.
+
+3. Follow the instructions provided by GitHub to complete the signup process.
+
+4. Verify your account by clicking on the confirmation link sent to your email.
+
+## Access Oppia's GitHub Repository
+
+1. Open your web browser and go to [Oppia's GitHub repository](https://github.com/oppia/oppia).
+
+2. You will see the main page of the Oppia repository where the development and issue tracking take place.
+
+## File a New Issue
+
+1. On the Oppia repository page, locate the "Issues" tab near the top navigation bar and click on it.
+![Screenshot showing link to the issue tab](images/githubIssueGuide/LinkToIssue.png)
+
+2. In the "Issues" tab, you will find a green button labeled "New issue." Click on it to create a new issue.
+![Screenshot showing link to new issue button](images/githubIssueGuide/LinkToNewIssue.png)
+
+3. Choose type of the issue which you want to file from the list of available types.
+![Screenshot showing types of issues](images/githubIssueGuide/typeOfIssues.png)
+
+4. On the issue creation page, provide a concise and descriptive title for your issue in the "Title" field and follow the steps as mentioned along with each comment box.
+
+   `Note :` Red starred comment box means it must be filled.
```

Comment:
Please don't use code blocks for emphasis. See the wiki style guide for details: https://github.com/oppia/oppia-web-developer-docs/blob/develop/Wiki-style-guide.md

Reply:
Done.